translations/validators.fr.yml

```
@@ -211,6 +211,8 @@ procuration.vote_country.conditions.fr: Attention, vous ne pouvez être mandatai
 procuration.vote_country.conditions.other: Attention, vous ne pouvez être mandataire que pour trois procurations dont une seule maximum établie en France.
 procuration.vote_country.conditions.conditions: Merci de bien vouloir remplir la condition ci-dessous.
 procuration.state.not_empty: Vous devez remplir l'État/Province si le pays de résidence n'est pas en France.
+procuration.postal_code.not_empty: Vous devez remplir le code postal si le pays de résidence est en France.
+procuration.vote_postal_code.not_empty: Vous devez remplir le code postal de votre lieu de vote si le pays de résidence est en France.
```

```suggestion
procuration.vote_postal_code.not_empty: Vous devez remplir le code postal de votre lieu de vote si le pays de résidence est France.
```
And change in tests.

---

Changed after a talk with our good Lord